src/main/java/io/cryostat/jmc/serialization/HyperlinkedSerializableRecordingDescriptor.java

```diff
@@ -85,6 +103,38 @@ public int hashCode() {
 
     @Override
     public boolean equals(Object o) {
```

Comment:
`equals` was updated here for the metadata field, but `hashCode` was not. `hashCode` and `equals` should always be consistent with each other - is that still preserved with this change?

Reply:
Oops, should be fixed now